app/src/main/res/raw/remote_config.json

```diff
@@ -2621,9 +4408,9 @@
           },
           "offset": 0,
           "opacity": 1,
-          "radius": 5
+          "radius": 1
         }
       }
     }
   }
-}
+}
```

Comment:
Config file is really huge.
Just an idea for future improvements, maybe it makes sense to split json config to several smaller files, per screen for example.

Reply:
This is temp file, I will remove it before release, but for testing purposes I agree, it would be better to split them.